Backend/tests/test_user_service.py

```diff
@@ -120,3 +128,8 @@ def public_key_1():
 @pytest.fixture
 def public_key_2():
     return '{"e":7,"n":3599}'
+
+
+@pytest.fixture
+def bu_service():
+    return UserService(DatabaseUser)
```

Comment:
This is just returning a class object (not an initialized instance of the class). You can use the already made `UserService` fixture that is in common fixtures (the other tests in this file already use it).